### PR TITLE
feat(meta): merge orm model to framework (Part I)

### PR DESCRIPTION
**Pull request by maxshuang**
close issue: #333 

feat(meta): merge orm model to framework

### CODE REVIEW

File: pkg/orm/util_test.go [hunk 1]
```diff
@@ -572,18 +574,18 @@ func TestJob(t *testing.T) {
...
-				// 'UPDATE `master_meta` SET `created_at`=?,`updated_at`=?,`project_id`=?,
+				// 'UPDATE `master_meta_kv_data` SET `created_at`=?,`updated_at`=?,`project_id`=?,
 				// `job_id`=?,`job_type`=?,`job_status`=?,`job_addr`=?,`job_config`=? WHERE `id` = ?'
 				fn: "UpdateJob",
 				inputs: []interface{}{
-					&model.MasterMeta{
+					&model.libModel.MasterMetaKVData{
 						Model: model.Model{
 							SeqID:        1,
 							CreatedAt: createdAt,
```
Comment (amyangfei) [inline pkg/orm/util_test.go:591]: Will this case be enabled later?

File: pkg/orm/util.go [hunk 2]
```diff
@@ -83,8 +85,8 @@ type MetaOpsClient struct {
 // Initialize will create all related tables in SQL backend
 // TODO: What if we change the definition of orm??
 func (c *MetaOpsClient) Initialize(ctx context.Context) error {
-	if err := c.db.AutoMigrate(&model.ProjectInfo{}, &model.ProjectOperation{}, &model.MasterMeta{},
-		&model.WorkerMeta{}, &model.ResourceMeta{}); err != nil {
```
Comment (amyangfei) [inline pkg/orm/util.go:87]: Can we remove `ResourceMeta` definition from `pkg/orm/model/resource.go` in this PR, ditto for several merged models.

File: pkg/orm/util_test.go (hunk 1 above)
Comment (maxshuang) [reply pkg/orm/util_test.go:591]: master_meta will be used later.

File: pkg/orm/util.go (hunk 2 above)
Comment (maxshuang) [reply pkg/orm/util.go:87]: ok

**Review by amyangfei — APPROVED** [review]

**Pull request merged**